include/libproject/libproject_error.h

```diff
@@ -19,16 +19,38 @@ class FileSetRuntimeError : public std::runtime_error {
     enum ErrType {
         UnknownError,
         IncorrectSource,
-        AlreadyLoaded,
-        NotLoaded,
-        BrokenSubproject,
+        ProjectAlreadyLoaded,
+        NotLoadedInterface,
+        AddBrokenSubproject,
+        LoadBrokenSubproject,
         SubprojectsIncongruity
     };
 
-    FileSetRuntimeError(const ErrType type, const std::string& message)
+    FileSetRuntimeError(const ErrType type, const std::string& message = std::string())
         : runtime_error(message), errorType(type) { }
 
     ErrType getErrorType() const { return errorType; }
+
+    const std::string toString() const
+    {
+        switch (errorType)
+        {
+        case UnknownError:
+            return std::string("Unknown Error\nSee additional info through what() function");
+        case IncorrectSource:
+            return std::string("Encountered problem with source\nSee additional info through what() function");
+        case ProjectAlreadyLoaded:
+            return std::string("Loader have loaded project already");
+        case NotLoadedInterface:
+            return std::string("Loader haven\'t loaded project");
+        case AddBrokenSubproject:
+            return std::string("There was try to add broken subproject");
+        case LoadBrokenSubproject:
+            return std::string("There was try to load broken subproject");
+        case SubprojectsIncongruity:
+            return std::string("Some incongruity detected\nSee additional info through what() function");
+        }
+    }
```

Comment:
@drewpts  yep, but it's unused as i seen
You just duplicate such info in constructors. It's wrong.
Exception shouldn't contain any text info. Only one case when it suitable - such info is based on client system input and can't be presented in less standartized and compact form.